integration/compatibility/generate_array.py

```
@@ -0,0 +1,252 @@
+        cmd = ["ft.aggregate", f"{key_type}_idx1", FILTER_QUERY]
+        cmd += ("load 5 @__key @n1 @n2 @t1 @t2 groupby 1 @t1 "
+                "reduce tolist 1 @n2 as items "
+                f"reduce tolist 1 @t2 as sitems {tail}").split()
+        self.execute_command(cmd + ["DIALECT", "2"])
+
+    ### APPLY ###
+
+    def test_apply_dyadic_ops(self, key_type):
+        self.setup_data(DATASET, key_type)
+        for op in DYADIC_OPS:
+            for l, r in DYADIC_OPERANDS:
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)